src/style.css

```
@@ -0,0 +1,143 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.product_card {
```

I don't see any block named product, so now you've created an element of not existing block, make this a block instead
```suggestion
.card {
```